src/components/ui/ReadyStyles/CircleButton.tsx

```diff
@@ -0,0 +1,25 @@
+import {StyleSheet, TouchableOpacity} from 'react-native';
+import React from 'react';
+
+type CircleButtonPropsType = {
+  children: React.ReactNode;
+  onPress: () => void;
+  disabled: boolean;
+};
+
+export const CircleButton = ({children, onPress, disabled}: CircleButtonPropsType) => {
```

Comment:
why is this called `CircleButton`? I just see `TouchableOpacity` wrapper with one style `opacity`